ldes-server-infra-mongo/mongock-changeset-4/README.md

```diff
@@ -0,0 +1,17 @@
+# Mongock Changeset 4
+
+This changeset updates the member id's by adding the collectionName as prefix.
+The prefix is separated by a '/'.
+This change makes memberIds unique per collection, allowing duplicate memberId's between different collections.
+
+The following configuration is needed to apply this changeset.
+
+```
+mongock:
+  migration-scan-package:
+    - be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset4
+```
+
+Consequences of Changeset 4:
+* LdesMembers:
+  * Changed field `_id`
```

Comment:
Less PR specific, but who will read this? :)

Reply:
They are the release-notes of the changeset. Anyone who wants to know what is in a changeset should read this.